lldb/Maintainers.md

```diff
@@ -0,0 +1,241 @@
+
+### Platforms
+
+The following people are responsible for decisions involving platforms.
+
+#### Android
+
+Pavel Labath
+pavel@labath.sk (email), [labath](https://github.com/labath) (GitHub), [labath](https://discourse.llvm.org/u/labath) (Discourse)
+
+#### Darwin
+
+Jim Ingham
+jingham@apple.com (email), [jimingham](https://github.com/jimingham) (GitHub), [jingham](https://discourse.llvm.org/u/jingham) (Discourse)
+
+Jason Molenda
+jmolenda@apple.com (email), [jasonmolenda](https://github.com/jasonmolenda) (GitHub), [jasonmolenda](https://discourse.llvm.org/u/jasonmolenda) (Discourse), jasonmolenda (Discord)
+
+Jonas Devlieghere
+jonas@devlieghere.com (email), [jdevlieghere](https://github.com/jdevlieghere) (GitHub), [jdevlieghere](https://discourse.llvm.org/u/jdevlieghere) (Discourse), jdevlieghere (Discord)
+
+#### FreeBSD
+
+Ed Maste
+emaste@freebsd.org (email), [emaste](https://github.com/emaste) (GitHub), [emaste](https://discourse.llvm.org/u/emaste) (Discourse), emaste (Discord)
+
+#### Linux
+
+Pavel Labath
+pavel@labath.sk (email), [labath](https://github.com/labath) (GitHub), [labath](https://discourse.llvm.org/u/labath) (Discourse)
+
+David Spickett
+david.spickett@linaro.org (email), [DavidSpickett](https://github.com/DavidSpickett) (GitHub), [DavidSpickett](https://discourse.llvm.org/u/DavidSpickett) (Discourse), davidspickett (Discord)
+
+#### Windows
+
+Omair Javaid
+omair.javaid@linaro.org (email), [omjavaid](https://github.com/omjavaid) (GitHub), [omjavaid](https://discourse.llvm.org/u/omjavaid) (Discourse), omjavaid#9902 (Discord)
+
+### Tools
+
+The following people are responsible for decisions involving specific tools.
+
+#### debugserver
+
+Jason Molenda
+jmolenda@apple.com (email), [jasonmolenda](https://github.com/jasonmolenda) (GitHub), [jasonmolenda](https://discourse.llvm.org/u/jasonmolenda) (Discourse), jasonmolenda (Discord)
+
+#### lldb-server
+
+David Spickett
+david.spickett@linaro.org (email), [DavidSpickett](https://github.com/DavidSpickett) (GitHub), [DavidSpickett](https://discourse.llvm.org/u/DavidSpickett) (Discourse), davidspickett (Discord)
+
+Pavel Labath
+pavel@labath.sk (email), [labath](https://github.com/labath) (GitHub), [labath](https://discourse.llvm.org/u/labath) (Discourse)
+
+#### lldb-dap
+
+Greg Clayton
+gclayton@fb.com (email), [clayborg](https://github.com/clayborg) (GitHub), [clayborg](https://discourse.llvm.org/u/clayborg) (Discourse)
+
+Walter Erquinigo
+a20012251@gmail.com (email), [walter-erquinigo](https://github.com/walter-erquinigo) (GitHub), [wallace](https://discourse.llvm.org/u/wallace) (Discourse), werquinigo (Discord)
+
+## Former Maintainers
+
+The following people have graciously spent time performing maintainership duties but are no longer active in that role. Thank you for all your help with the success of the project!
+
+Kamil Rytarowski (kamil@netbsd.org)
```

Comment:
These are rendering as one line, I think they need a blank line in between to make it separate.